webapp/loggedin/modules/users/user/components/profilePictureEditor.js

```diff
@@ -0,0 +1,95 @@
+import './profilePictureEditor.scss'
+
+import React, { useEffect, useRef, useState } from 'react'
+import AvatarEditor from 'react-avatar-editor'
+
+import { useFileDrop } from '../../../../../commonComponents/hooks'
+
+const ProfilePictureEditor = ({ image: initialImage, onPictureUpdate, enabled }) => {
+
+  const dropRef = useRef(null)
+  const avatarRef = useRef(null)
+
+  const [image, setImage] = useState(null)
+  const [scale, setScale] = useState(1)
+  const [rotate, setRotate] = useState(0)
+
+  useEffect(() => {
+    if (initialImage) {
+      setImage(initialImage)
+    }
+  }, [initialImage])
+
+  useFileDrop(dropRef, files => {
+    const imgFile = files.find(f => f.kind === 'file' && f.type.startsWith('image'))
```

Comment:
why don't you pass the accepted file types to the hook directly?

Reply:
Ok, passing a list of RegExp, and returning just the file